oshmem/mca/memheap/base/base.h

```diff
@@ -188,6 +188,26 @@ static inline int memheap_find_segnum(void *va)
     return MEMHEAP_SEG_INVALID;
 }
 
+static inline int memheap_find_remote_segnum(void *va, int pe)
```

Comment:
No need for inline for non-data path 
Can we just avoid the function?

Reply:
Sure, we can include it in rmkey_free body as it has only one usage inside rmkey_free as of now. I put it as inline as the memheap_find_segnum is inline. However, memheap_find_segnum is used in several places.